src/GEM_u8g2.cpp

```diff
@@ -358,18 +371,16 @@ void GEM_u8g2::printMenuItems() {
   byte y = _menuPageScreenTopOffset;
   byte i = 0;
   while (menuItemTmp != 0 && i < _menuItemsPerScreen) {
-    byte yText = y + getMenuItemInsetOffset();
-    byte yDraw = y + getMenuItemInsetOffset(true);
+    byte yText = y + getMenuItemInsetOffset()-3;
```

Comment:
If we change this to:

```suggestion
    byte yText = y + getMenuItemInsetOffset()-2;
```

then the other changes of offsets in line 459 and 647 can be reverted.